great_expectations/data_context/store/inline_store_backend.py

```diff
@@ -183,8 +184,14 @@ def _has_key(self, key: Tuple[str, ...]) -> bool:
         return resource_type in self._data_context.config
 
     def _save_changes(self) -> None:
-        # NOTE: <DataContextRefactor> This responsibility will be moved into DataContext Variables object
-        self._data_context._save_project_config()
+        context = self._data_context
+        config_filepath = os.path.join(context.root_directory, context.GE_YML)
```

Comment:
Can we use `pathlib` for this?
```suggestion
        config_filepath = pathlib.Path(context.root_directory) / context.GE_YML
```

Reply:
done!